specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,384 @@
+						}
+					}
+				},
+				"x-ms-examples": {
+					"CosmosDBNotebookWorkspaceGet": {
+						"$ref": "./examples/CosmosDBNotebookWorkspaceGet.json"
+					}
+				}
+			},
+			"put": {
+				"tags": [
+					"NotebookWorkspacesResource"
+				],
+				"description": "Creates the notebook workspace for a Cosmos DB account.",
+				"operationId": "NotebookWorkspaces_Create",
+				"x-ms-long-running-operation": true,
+				"parameters": [{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+					},
+					{
+						"$ref": "#/parameters/NotebookWorkspaceNameParameter"
+					},
+					{
+						"name": "createParameters",
+						"in": "body",
+						"required": true,
+						"schema": {
+							"$ref": "#/definitions/NotebookWorkspaceCreateParameters"
+						},
+						"description": "The notebook workspace to create for the current database account."
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Creation of notebook workspace will complete asynchronously.",
+						"schema": {
+							"$ref": "#/definitions/NotebookWorkspace"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"CosmosDBNotebookWorkspaceCreate": {
+						"$ref": "./examples/CosmosDBNotebookWorkspaceCreate.json"
+					}
+				}
+			},
+			"delete": {
+				"tags": [
+					"NotebookWorkspacesResource"
+				],
+				"description": "Deletes the notebook workspace for a Cosmos DB account.",
+				"operationId": "NotebookWorkspaces_Delete",
+				"x-ms-long-running-operation": true,
+				"parameters": [{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+					},
+					{
+						"$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+					},
+					{
+						"$ref": "#/parameters/NotebookWorkspaceNameParameter"
+					}
+				],
+				"responses": {
+					"202": {
+						"description": "The notebook workspace delete operation will complete asynchronously."
+					},
+					"204": {
+						"description": "The specified notebook workspace does not exist in the database account."
+					}
+				},
+				"x-ms-examples": {
+					"CosmosDBNotebookWorkspaceDelete": {
+						"$ref": "./examples/CosmosDBNotebookWorkspaceDelete.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/notebookWorkspaces/{notebookWorkspaceName}/getConnectionInfo": {
```

Comment:
"list" is a preferred prefix for POST actions that return connection strings/secrets. Actions with "list" as the prefix verb can be called from an ARM template

Reply:
Thanks for the feedback. Are you saying actions MUST have a "list" prefix to be usable in ARM template? If so, we will make this change.

Reply:
Yep

Reply:
The change has been deployed.